pkg/core/localize/locales/en/cmd/rule.en.toml

```diff
@@ -70,13 +71,13 @@ one='Group:'
 one='Artifact group. Leave blank for global rule'
 
 [registry.rule.common.error.invalidRuleConfig]
-one = 'invalid configuration value "{{.Config}}" for rule type "{{.RuleType}}". Valid operations are: {{.ValidConfigList}}'
+one = 'Invalid configuration value "{{.Config}}" for rule type "{{.RuleType}}". Valid configuration values are as follows: {{.ValidConfigList}}'
```

Comment:
The CLI starts error messages with lowercase as of now. Is there any reason behind keeping it uppercase?

Reply:
I think it's better practice to start a new message with an uppercase letter, because if you show multiple consecutive messages and some of them are multi-line, the uppercase starting letter makes it easier to distinguish between the messages and makes them easier to read. However, if you are already using a convention of starting with lowercase, we can continue that practice for now for consistency. We can investigate starting with uppercase (in all messages) in a separate PR, if we decide to change the convention.

Reply:
I will agree on the multiple consecutive messages, ignore this suggestion.